pkg/runtime/runtime_test.go

```diff
@@ -9,13 +9,13 @@ import (
 func TestNew(t *testing.T) {
 	t.Parallel()
 	rt := runtime.New()
-	if rt == nil {
+	if rt == nil { //nolint:staticcheck
 		t.Fatal("runtime must not be nil")
 	}
-	if rt.GOOS == "" {
+	if rt.GOOS == "" { //nolint:staticcheck
 		t.Fatal("rt.GOOS is empty")
 	}
-	if rt.GOARCH == "" {
+	if rt.GOARCH == "" { //nolint:staticcheck
```

Comment:
I noticed some lint errors here, so I've added nolint annotations. lmk what you think

Reply:
Weird... test-ci is currently failing on these annotations but running local does not give me any errors 🤔